src/openrct2-ui/interface/Window.cpp

```diff
@@ -681,6 +688,21 @@ void InvalidateAllWindowsAfterInput()
     });
 }
 
+bool Window::IsLegacy()
```

Comment:
What does "legacy" mean in this context?

Reply:
If it has been converted to a class or if it's the old style rct_window